.github/workflows/ci.yml

```diff
@@ -165,19 +165,26 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.BACKPACK_DOCS_DEPLOY_TOKEN }}
 
-#      No need for now - need to create AppCenter app first
-#      - name: Building release APK
-#        run: |
-#          openssl aes-256-cbc -K $encrypted_8363dc54ada0_key -iv $encrypted_8363dc54ada0_iv -in backpack.android.demo.key.enc -out backpack.android.demo.key -d;
-#          ./gradlew :app:assemble${{ env.flavour }}Release -PdisablePreDex
-#
-#      - name: Publish APK to AppCenter
-#        if: github.ref == 'refs/heads/main'
-#        uses: wzieba/AppCenter-Github-Action@v1
-#          with:
-#            appName: Skyscanner/Backpack-Android
-#            token: ${{secrets.APP_CENTER_TOKEN}}
-#            group: Testers
-#            file: app/build/outputs/apk/oss/release/app-oss-release.apk
-#            notifyTesters: false
-#            debug: false
+      - name: Building release APK
+        if: contains('refs/tags/', github.ref)
```

Comment:
Should we also add an && condition to make sure this is when its on main branch too? as this looks to essentially say run if ref contains tags which could happen on any commit or PR that also includes any tags?

Or does this also push PR build versions too?